chpl/chpl-service/src/main/java/gov/healthit/chpl/dto/CertificationCriterionDTO.java

```diff
@@ -9,8 +9,10 @@
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.NoArgsConstructor;
+import lombok.ToString;
 
 @Builder
+@ToString
```

Comment:
Is there a reason _not_ to use the `@Data` annotation here instead, and remove the various Getters and Setters?